pkg/policy/l4.go

```diff
@@ -321,13 +321,13 @@ func NewL4Policy() *L4Policy {
 	}
 }
 
-// IngressCoversContext checks if the receiver's ingress `L4Policy` contains
+// IngressCoversContext checks if the receiver's ingress `RealizedL4Policy` contains
```

Comment:
I'm not sure why this was single-backquoted before, but as far as I can tell the receiver type doesn't have anything to do with realized/desired state, it's just a representation of a map type which could be used as either realized/desired. Does this comment update make sense? (Also for `EgressCoversContext`)